lib/chat/chat.dart

```diff
@@ -1,18 +1,78 @@
+import 'package:google_fonts/google_fonts.dart';
 import 'package:flutter/material.dart';
+import './new_matches.dart';
+import './messages.dart';
 
 // TODO: Update this placeholder widget
 class Chat extends StatelessWidget {
   const Chat({Key key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-      backgroundColor: Colors.black,
-      body: Container(
-        child: Center(
-          child: Text("Matches", style: TextStyle(color: Colors.white, fontSize: 30.0))
-        )
-      )
+    return Stack(
+      children: <Widget>[
+        Image.asset(
+          "images/DefaultBg.png",
+          height: MediaQuery.of(context).size.height,
+          width: MediaQuery.of(context).size.width,
+          fit: BoxFit.fill,
+        ),
+        Scaffold(
+          backgroundColor: Colors.white,
+          appBar: AppBar(
+            backgroundColor: Colors.transparent,
+            elevation: 0.0,
+          ),
```

Comment:
I think you can get rid of this `appBar` class right here since you are building your own header! `appBar` is basically a title bar for the page and if you remove it, it will also remove some of the empty space that you have on top of your header. You can read more about it here: https://api.flutter.dev/flutter/material/AppBar-class.html

It would look like this without it, but you could always add padding/margin on your header widget! 
<img src="https://user-images.githubusercontent.com/25041263/82510619-47629080-9ac8-11ea-889d-0b30d8155936.png" height="537.6" width="248.4">